app-backend/batch/src/main/scala/stacExport/WriteStacCatalog.scala

```diff
@@ -229,7 +232,7 @@ final case class WriteStacCatalog(
       _ = logger.info(s"Writing export under prefix: $exportPath")
       catalog = Utils.getAnnotationProjectStacCatalog(
         exportDefinition,
-        "1.0.0-beta.1",
+        "1.0.0",
```

Comment:
> This makes me think if it is possible to have this version number somewhere so that we only update it once every time we do upgrades, instead of tracking them down like this... What do you think?

This would be a better thing to do definitely. It's been a few days since I made this change, but I think there was a reason I didn't do this this time -- maybe it's used in different `objects` in different scopes? I'll have a look and see if that's right / if I can fix it by just adding a `STACVersion` object with a `version: String` attribute or something.

> can different stac item in the same collection and/or catalog comply to different versions of the spec?

There's no rule against it. We only support one at a time, like, we can't decode both a v0.9.x item and a 1.0.x item in teh same version of stac4s. But it's technically allowed.